test/gapic-mocks/runQuery.ts

```diff
@@ -68,35 +53,13 @@ describe('Run Query', () => {
     }
   }
 
-  it('should pass read time into runQuery for transactions', async () => {
```

Comment:
Note that this test is no longer valid because transaction and read time are no longer compatible together as is the case in Python.